user_service_basic.go

```diff
@@ -0,0 +1,83 @@
+package gimlet
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// BasicUserManager implements the UserManager interface and has a list of
+// BasicUsers which is stored in the settings configuration file.
+// Note: This use of the UserManager is recommended for dev/test purposes only
+// and users who need high security authentication  mechanisms should rely on a
+// different authentication mechanism.
+type BasicUserManager struct {
+	users []basicUser
+}
+
+func NewBasicUserManager(users []basicUser) (UserManager, error) {
```

Comment:
should just have a slice of users, and do type conversion here.

also docstring.